ui/components/app/add-network/index.scss

```
@@ -8,10 +8,13 @@
     }
   }
 
-  &__token-image {
-    margin-right: 7px;
-    height: 24px;
-    width: 24px;
+  &__warning-tooltip {
+    color: var(--ui-4);
+    width: 180px;
+
+    a {
+      color: var(--primary-1);
```

Update to new design token color css variable
```suggestion
      color: var(--color-primary-default);
```

---

Done